.mergify.yml

```diff
@@ -0,0 +1,44 @@
+queue_rules:
+  - name: default
+    conditions:
+      - "#approved-reviews-by>=1"
+      - "#changes-requested-reviews-by=0"
+      - status-success=Lint
+      - status-success=Python (3.8, macos-latest)
```

Comment:
`tsdate` doesn't have github actions tests, so these merge conditions will never be met. 
You just need:

      - "status-success=ci/circleci: build"
      - status-success=Docs